aws_adfs/login.py

```diff
@@ -231,12 +231,13 @@ def login(
         _emit_summary(config, aws_session_duration)
 
 
-def _emit_json(aws_session_token):
+def _emit_json(aws_session_token, aws_session_duration):
```

Comment:
[datetime.timezone is new in python version 3.2](https://docs.python.org/3/library/datetime.html#datetime.timezone) so we need to implement our own `tzinfo` subclass to maintain python 2 compatibility. Code inspired from examples at https://docs.python.org/2/library/datetime.html#datetime.tzinfo.fromutc

An alternative maybe to use [pytz](https://git.launchpad.net/pytz) but dragging another dependency just for this is probably overkill.

```suggestion
def _emit_json(aws_session_token, aws_session_duration):
    class UTC(tzinfo):
        def utcoffset(self, dt):
            return datetime.timedelta(0)

        def tzname(self, dt):
            return "UTC"

        def dst(self, dt):
            return datetime.timedelta(0)

```